### PR TITLE
Module 2 day 1 farooq dev

### PR DESCRIPTION
**Pull request by farooqgollapallishaik**

### CODE REVIEW

**Comment by farooqgollapallishaik** [comment]
Taks.txt is deleted form Task , new Task folder is created in that Farooq_task.html file is added, assignment is added in Farooq_task.html file. 
Cheat Sheet is Modified.

File: Module-2/Day-1/Task/Farooq/Farooq_task.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <script>
+        let admin, name;
+            name = "John";
+            admin = name;
+            alert(admin);
+
```
Comment (IshaaqhShaik) [inline Module-2/Day-1/Task/Farooq/Farooq_task.html:14]: create a separate MR for these changes

**Review by IshaaqhShaik — CHANGES_REQUESTED** [review]

**Review by IshaaqhShaik — CHANGES_REQUESTED** [review]
please isolate the cheatsheet and task to respective MR's

**Review by IshaaqhShaik — CHANGES_REQUESTED** [review]
please isolate the cheatsheet and task to respective MR's

**Pull request closed without merging**